acceptance-components/react-app/styles.css

```diff
@@ -7,4 +7,23 @@
 .special {
   background: palevioletred;
   color: white;
-}
\ No newline at end of file
+}
+
+/*
+ * to test postcss (postcss-icss-values) plugin
+ * this should return:
+ *
+ *  .superSpecial {
+ *    color: blue;
+ *  }
+ *
+ *  -- colors.css --
+ *
+ *  @value bgColor: blue;
+ *
+ */
+@value bgColor from './colors.css';
```

Comment:
Thanks 💯  for adding this to the tests!
I guess we'll also need to check in the `color.css file` or am I missing something?